cmd/servicelog/common.go

```diff
@@ -15,6 +15,7 @@ import (
 var (
 	templateParams, userParameterNames, userParameterValues, filterParams []string
 	HTMLBody                                                              []byte
+	responseMessage                                                       string
```

Comment:
Why using a global variable for this?

Reply:
Thank you for your comments.
Good point, I've eliminated the use of global variable